package/src/method-element.js

```diff
@@ -86,11 +86,23 @@ class MethodElement {
     return options
   }
 
+  /**
+   * Read any cssSrc includes for font loading from the settings.
+   */
+  getCssSrc () {
+    return this.settings.css_src.reduce((acc, item) => {
+      if (typeof item === 'string') {
```

Comment:
Why is this needed?

Reply:
I guess because Stripe’s JS accepts either an URL to a CSS file for the key `cssSrc` or an object containing the font-face declaration (see [doc](https://stripe.com/docs/stripe-js/reference#elements-options)). That’s kind of what the comment says, too, although not very clearly ;)